app/components/Navigation.js

```diff
@@ -96,28 +88,28 @@ class Navigation extends Component {
     const { user } = this.props;
     return (
       <Navbar
-        bg="dark"
+        id="navbar"
         style={{
           height: '7rem',
-          backgroundColor: 'black',
+          backgroundColor: '#070707',
           fontWeight: '100',
           padding: '0 5rem'
         }}
       >
-        <Navbar.Brand>Logo</Navbar.Brand>
+        <Navbar.Brand id="logo">Climbar</Navbar.Brand>
```

Comment:
Can we make the brand size a little bit bigger?

Reply:
Done